NuRadioMC/simulation/simulation.py

```diff
@@ -6,60 +6,56 @@
-import NuRadioReco.modules.electricFieldResampler
-import NuRadioReco.modules.channelGenericNoiseAdder
-import NuRadioReco.modules.efieldToVoltageConverterPerEfield
-import NuRadioReco.modules.efieldToVoltageConverter
-import NuRadioReco.modules.channelAddCableDelay
-import NuRadioReco.modules.channelResampler
-import NuRadioReco.modules.triggerTimeAdjuster
-from NuRadioReco.detector import detector
+
+from NuRadioReco.modules.io import eventWriter
+from NuRadioReco.modules import (
+    channelAddCableDelay, electricFieldResampler, efieldToVoltageConverterPerEfield,
+    efieldToVoltageConverter, channelSignalReconstructor, channelResampler,
+    channelGenericNoiseAdder, triggerTimeAdjuster)
+
+from NuRadioReco.detector import detector, antennapattern
 import NuRadioReco.framework.sim_station
 import NuRadioReco.framework.electric_field
 import NuRadioReco.framework.particle
 import NuRadioReco.framework.event
 import NuRadioReco.framework.sim_emitter
-from NuRadioReco.detector import antennapattern
-from NuRadioReco.framework.parameters import electricFieldParameters as efp
-from NuRadioReco.framework.parameters import showerParameters as shp
-from NuRadioReco.framework.parameters import channelParameters as chp
-# parameters describing simulated Monte Carlo particles
-from NuRadioReco.framework.parameters import particleParameters as simp
-from NuRadioReco.framework.parameters import emitterParameters as ep
-# parameters set in the event generator
-from NuRadioReco.framework.parameters import generatorAttributes as genattrs
+
+from NuRadioReco.framework.parameters import (
+    electricFieldParameters as efp, showerParameters as shp,
+    channelParameters as chp, particleParameters as simp, emitterParameters as ep,
+    generatorAttributes as genattrs)
+
 import NuRadioMC.simulation.time_logger
 
 logger = logging.getLogger("NuRadioMC.simulation")
 
+
 # initialize a few NuRadioReco modules
 # TODO: Is this the best way to do it? Better to initialize them on demand
-channelAddCableDelay = NuRadioReco.modules.channelAddCableDelay.channelAddCableDelay()
-electricFieldResampler = NuRadioReco.modules.electricFieldResampler.electricFieldResampler()
-efieldToVoltageConverterPerEfield = NuRadioReco.modules.efieldToVoltageConverterPerEfield.efieldToVoltageConverterPerEfield()
-efieldToVoltageConverter = NuRadioReco.modules.efieldToVoltageConverter.efieldToVoltageConverter()
-channelSignalReconstructor = NuRadioReco.modules.channelSignalReconstructor.channelSignalReconstructor()
-channelResampler = NuRadioReco.modules.channelResampler.channelResampler()
-channelGenericNoiseAdder = NuRadioReco.modules.channelGenericNoiseAdder.channelGenericNoiseAdder()
-channelResampler = NuRadioReco.modules.channelResampler.channelResampler()
-eventWriter = NuRadioReco.modules.io.eventWriter.eventWriter()
-triggerTimeAdjuster = NuRadioReco.modules.triggerTimeAdjuster.triggerTimeAdjuster()
+channel_add_cable_delay = channelAddCableDelay.channelAddCableDelay()
```

Comment:
please don't do that. Now the module name is changed. This will make the code less readable as it is unclear if we have a variable, a custom function or a NuRadio module.

Reply:
I think having a object named the same name as a class is not a good idea neither. Do you want to keep a certain syntax for class objects?

Reply:
I have seen many occasions where this lead to the fact that the module/class is overwritten by a object

Reply:
if you do the import like you refactored it to, yes. But even then, this is rarely a problem as the module needs to be instantiated only once. 

Throughout NuRadio, the module instances are named exactly the same as the class name. So this would constitute a major change in the coding style. And I would find it less intuitive. Whenever you see camelCase, you know that it is a NuRadio module.

Reply:
okay

Reply:
I reverted this